src/main/java/core/basesyntax/io/read/impl/csv/CsvReaderImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.io.read.impl.csv;
+
+import core.basesyntax.io.read.ReportReader;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class CsvReaderImpl implements ReportReader {
+    private final String reportPath;
+
+    public CsvReaderImpl(String reportPath) {
+        this.reportPath = reportPath;
+    }
+
+    public String readReport() throws FileNotFoundException {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(reportPath))) {
+            if (reportPath.isEmpty()) {
+                throw new FileNotFoundException("The path specified is incorrect");
+            }
+            StringBuilder sb = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                sb.append(value);
```

Comment:
When appending the lines from the file to the `StringBuilder`, consider adding a line separator to preserve the original formatting of the CSV file. For example: `sb.append(value).append(System.lineSeparator());`